src/Client.php

```diff
@@ -326,9 +326,7 @@ public function waitFor(string $locator, int $timeoutInSecond = 30, int $interva
     {
         $locator = trim($locator);
```

Comment:
```suggestion
```

`trim()` is already called in `createWebDriverByFromLocator()`

Reply:
Removed, thanks